src/api/custom-resolvers/index.ts

```diff
@@ -11,7 +11,7 @@ import {
   deleteApplicationByReferenceNumber,
   updateCompanyAndCompanyAddress,
   submitApplication,
-  sendEmailInsuranceFeedback,
+  createInsuranceFeedbackAndEmail,
```

Comment:
How about just `createFeedback` or `createInsuranceFeedback`?

Will the quote feedback data and email be any different?

Reply:
I am not sure yet, it may have extra fields from the design

Reply:
it cannot be createFeedback as that clashes with the autogenerated resolver by keystone.  Set to `createFeedbackAndEmail` so is generic

Reply:
OK great, `createFeedbackAndSendEmail` would be ideal, not a blocker